libfoundation/include/foundation.h

```diff
@@ -1422,7 +1422,8 @@ extern MCStringRef kMCTabString;
 // the c-string must be a C static string.
 MCStringRef MCSTR(const char *string);
 
-const char *MCStringGetCString(MCStringRef p_string);
+// SN-2015-05-014: [[ MCStringGetCString Removal ]] End of the evil function.
+//const char *MCStringGetCString(MCStringRef p_string);
```

Comment:
Don't bother commenting about things that are removed - just delete them... The commit log should indicate what work was done (which can be seen via blame).